### DISCUSSION TITLE
Having issues when trying to install [all_extras] option

### DISCUSSION THREAD

**Discussion by larson-7** [alan-turing-institute/sktime#2071] [Q&A] [answered]
I can install the normal version no problem, but I'd like to install the full version and have been unsuccessful.

My install command:
 "pip3 install sktime[all_extras]"

Response:
"no matches found: sktime[all_extras]"

Any suggestions as to what I'm doing wrong?

**Comment by lmmentel** [comment, marked as answer]
Hi @larson-7!  Would you mind sharing what OS and python version are you on? You might also want to try

```bash
pip3 install "sktime[all_extras]"
```

and see if that works.
